I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00action_file.rpy", line 499, in __call__
    renpy.load(fn)
Exception: Couldn't find a place to stop rolling back. Perhaps the script changed in an incompatible way?

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/common/_layout/screen_main_menu.rpym", line 28, in script
    python hide:
  File "C:\Users\kansa\Desktop\renpy-8.1.3-sdk\renpy\ast.py", line 834, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\kansa\Desktop\renpy-8.1.3-sdk\renpy\python.py", line 1184, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "renpy/common/_layout/screen_main_menu.rpym", line 28, in <module>
    python hide:
  File "renpy/common/_layout/screen_main_menu.rpym", line 35, in _execute_python_hide
    ui.interact()
  File "C:\Users\kansa\Desktop\renpy-8.1.3-sdk\renpy\ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "C:\Users\kansa\Desktop\renpy-8.1.3-sdk\renpy\display\core.py", line 2218, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "C:\Users\kansa\Desktop\renpy-8.1.3-sdk\renpy\display\core.py", line 3289, in interact_core
    rv = root_widget.event(ev, x, y, 0)
  File "C:\Users\kansa\Desktop\renpy-8.1.3-sdk\renpy\display\layout.py", line 1297, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "C:\Users\kansa\Desktop\renpy-8.1.3-sdk\renpy\display\layout.py", line 1297, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "C:\Users\kansa\Desktop\renpy-8.1.3-sdk\renpy\display\layout.py", line 1297, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "C:\Users\kansa\Desktop\renpy-8.1.3-sdk\renpy\display\screen.py", line 794, in event
    rv = self.child.event(ev, x, y, st)
  File "C:\Users\kansa\Desktop\renpy-8.1.3-sdk\renpy\display\layout.py", line 1297, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "C:\Users\kansa\Desktop\renpy-8.1.3-sdk\renpy\display\layout.py", line 1526, in event
    rv = super(Window, self).event(ev, x, y, st)
  File "C:\Users\kansa\Desktop\renpy-8.1.3-sdk\renpy\display\layout.py", line 285, in event
    rv = d.event(ev, x - xo, y - yo, st)
  File "C:\Users\kansa\Desktop\renpy-8.1.3-sdk\renpy\display\layout.py", line 1297, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "C:\Users\kansa\Desktop\renpy-8.1.3-sdk\renpy\display\layout.py", line 1526, in event
    rv = super(Window, self).event(ev, x, y, st)
  File "C:\Users\kansa\Desktop\renpy-8.1.3-sdk\renpy\display\layout.py", line 285, in event
    rv = d.event(ev, x - xo, y - yo, st)
  File "C:\Users\kansa\Desktop\renpy-8.1.3-sdk\renpy\display\layout.py", line 1297, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "C:\Users\kansa\Desktop\renpy-8.1.3-sdk\renpy\display\layout.py", line 285, in event
    rv = d.event(ev, x - xo, y - yo, st)
  File "C:\Users\kansa\Desktop\renpy-8.1.3-sdk\renpy\display\viewport.py", line 589, in event
    rv = super(Viewport, self).event(ev, x, y, st)
  File "C:\Users\kansa\Desktop\renpy-8.1.3-sdk\renpy\display\layout.py", line 285, in event
    rv = d.event(ev, x - xo, y - yo, st)
  File "C:\Users\kansa\Desktop\renpy-8.1.3-sdk\renpy\display\layout.py", line 1297, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "C:\Users\kansa\Desktop\renpy-8.1.3-sdk\renpy\display\behavior.py", line 1182, in event
    return handle_click(self.clicked)
  File "C:\Users\kansa\Desktop\renpy-8.1.3-sdk\renpy\display\behavior.py", line 1103, in handle_click
    rv = run(action)
  File "C:\Users\kansa\Desktop\renpy-8.1.3-sdk\renpy\display\behavior.py", line 401, in run
    return action(*args, **kwargs)
  File "renpy/common/00action_file.rpy", line 499, in __call__
    renpy.load(fn)
  File "C:\Users\kansa\Desktop\renpy-8.1.3-sdk\renpy\loadsave.py", line 838, in load
    log.unfreeze(roots, label="_after_load")
  File "C:\Users\kansa\Desktop\renpy-8.1.3-sdk\renpy\rollback.py", line 1135, in unfreeze
    self.rollback(0, force=True, label=label, greedy=greedy, on_load=True)
  File "C:\Users\kansa\Desktop\renpy-8.1.3-sdk\renpy\rollback.py", line 930, in rollback
    self.load_failed()
  File "C:\Users\kansa\Desktop\renpy-8.1.3-sdk\renpy\rollback.py", line 848, in load_failed
    raise Exception("Couldn't find a place to stop rolling back. Perhaps the script changed in an incompatible way?")
Exception: Couldn't find a place to stop rolling back. Perhaps the script changed in an incompatible way?

Windows-10-10.0.19045 AMD64
Ren'Py 8.3.6.25022803
HeartsDepthDEMO 2.5
Wed Sep 17 23:18:48 2025
